CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Added an option to enable the light theme.
```

Comment:
Please add a section according to https://keepachangelog.com/en/1.1.0/

I would suggest `Added`

Reply:
done